### ISSUE TITLE
Sample for </td> is repetitive and incorrect.

### ISSUE THREAD

**Issue by Pankaj-Thakur** [MicrosoftDocs/dynamics-365-unified-operations-public#840]
Sample for &lt;/td&gt; is repetitive and incorrect.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 27a2af11-bc8e-9ffd-802c-12b3a91a739a
* Version Independent ID: dd23ab4f-2e5f-b99f-deae-28df175462ba
* Content: [X++ string runtime functions - Finance & Operations | Dynamics 365 | #MSDyn365FO](https://docs.microsoft.com/en-us/dynamics365/unified-operations/dev-itpro/dev-ref/xpp-string-run-time-functions)
* Content Source: [articles/dev-itpro/dev-ref/xpp-string-run-time-functions.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/live/articles/dev-itpro/dev-ref/xpp-string-run-time-functions.md)
* Service: **dynamics-ax-platform**
* GitHub Login: @RobinARH
* Microsoft Alias: **robinr**

**Comment by tonyafehr** [comment]
@Pankaj-Thakur, thanks for the feedback.  

@robadawy, can you update this topic to make this a better example.

**Comment by tonyafehr** [comment]
Bug 288975 has been logged to track this issue.

**Comment by legunter** [comment]
I just noticed this as well. The issue is that the </td> section under match should be describing the backslash (\) character, but seems to not display properly or is incorrect. This should be updated.

Additionally, it is required to have the @ symbol before strings containing the backslash for it to work properly, 

for example I needed to make a match string which matched either open and close brackes [ and ]. To do this would look like: '[\[\]]', but when debugging the xpp runtime removed the slashes and just made the string '[[]]' without the slashes when passing into the function. I needed to add @ symbol to keep the backslashes in the string like: @'[\[\]]'
